tests/lib/rules/one-var.js

```diff
@@ -1237,8 +1354,8 @@ ruleTester.run("one-var", rule, {
 			],
 		},
 		{
-			code: "var f,          /* test */ l;",
-			output: "var f;          /* test */ var l;",
+			code: "var f,         /* test */ l;",
+			output: "var f;         /* test */ var l;",
```

Comment:
```suggestion
			code: "var f,          /* test */ l;",
			output: "var f;          /* test */ var l;",
```

To avoid unnecessary diff.